magenta-lib/src/main/scala/magenta/deployment_type/BrazePublish.scala

```diff
@@ -0,0 +1,53 @@
+package magenta.deployment_type
+
+import play.api.libs.json.{JsArray, JsString, JsBoolean}
+
+object BrazePublish extends LambdaInvoke {
+  val brazePublishLambdaNameMinusStage =
+    s"${LambdaInvoke.lambdaFunctionNamePrefix}braze-version-control-publish-lambda-"
+  override val name = "braze-publish"
+  private val summary =
+    s"Invokes Lambda `${brazePublishLambdaNameMinusStage}` (with the STAGE appended). This Lambda publishes content blocks to the target Braze environment (only if there are changes)."
```

Comment:
Does this mean we're invoking a lambda by name and _not_ via tag lookup? If so, is that desirable? (I'd suggest it isn't).